examples/bento/test-page.html

```diff
@@ -0,0 +1,472 @@
+    custom-element="bento-inline-gallery"
+    src="https://cdn.ampproject.org/v0/bento-inline-gallery-1.0.js"
+  ></script>
+  <script
+    async
+    custom-element="bento-instagram"
+    src="https://cdn.ampproject.org/v0/bento-instagram-1.0.js"
+  ></script>
+  <script
+    async
+    custom-element="bento-jwplayer"
+    src="https://cdn.ampproject.org/v0/bento-jwplayer-1.0.js"
+  ></script>
+  <script
+    async
+    custom-element="bento-lightbox"
+    src="https://cdn.ampproject.org/v0/bento-lightbox-1.0.js"
+  ></script>
+  <script
+    async
+    custom-element="bento-lightbox-gallery"
+    src="https://cdn.ampproject.org/v0/bento-lightbox-gallery-1.0.js"
+  ></script>
+  <script
+    async
+    custom-element="bento-mathml"
+    src="https://cdn.ampproject.org/v0/bento-mathml-1.0.js"
+  ></script>
+  <script
+    async
+    custom-element="bento-selector"
+    src="https://cdn.ampproject.org/v0/bento-selector-1.0.js"
+  ></script>
+  <script
+    async
+    custom-element="bento-sidebar"
+    src="https://cdn.ampproject.org/v0/bento-sidebar-1.0.js"
+  ></script>
+  <script
+    async
+    custom-element="bento-social-share"
+    src="https://cdn.ampproject.org/v0/bento-social-share-1.0.js"
+  ></script>
+  <script
+    async
+    custom-element="bento-soundcloud"
+    src="https://cdn.ampproject.org/v0/bento-soundcloud-1.0.js"
+  ></script>
+  <script
+    async
+    custom-element="bento-stream-gallery"
+    src="https://cdn.ampproject.org/v0/bento-stream-gallery-1.0.js"
+  ></script>
+  <script
+    async
+    custom-element="bento-timeago"
+    src="https://cdn.ampproject.org/v0/bento-timeago-1.0.js"
+  ></script>
+  <script
+    async
+    custom-element="bento-twitter"
+    src="https://cdn.ampproject.org/v0/bento-twitter-1.0.js"
+  ></script>
+  <script
+    async
+    custom-element="bento-video"
+    src="https://cdn.ampproject.org/v0/bento-video-1.0.js"
+  ></script>
+  <script
+    async
+    custom-element="bento-video-iframe"
+    src="https://cdn.ampproject.org/v0/bento-video-iframe-1.0.js"
+  ></script>
+  <script
+    async
+    custom-element="bento-vimeo"
+    src="https://cdn.ampproject.org/v0/bento-vimeo-1.0.js"
+  ></script>
+  <script
+    async
+    custom-element="bento-wordpress-embed"
+    src="https://cdn.ampproject.org/v0/bento-wordpress-embed-1.0.js"
+  ></script>
+  <script
+    async
+    custom-element="bento-youtube"
+    src="https://cdn.ampproject.org/v0/bento-youtube-1.0.js"
+  ></script>
+</head>
+<body>
+    <div>
+    <h1>Bento Accordion</h1>
+  <bento-accordion id="my-accordion" disable-session-states>
+    <section>
+      <h2>Section 1</h2>
+      <p>Content in section 1.</p>
+    </section>
+    <section>
+      <h2>Section 2</h2>
+      <div>Content in section 2.</div>
+    </section>
+    <section expanded>
+      <h2>Section 3</h2>
+      <div>Content in section 3.</div>
+    </section>
+  </bento-accordion>
+    </div>
+<!--
+  <div>
+  <h1>Bento Brightcove</h1>
+<bento-brightcove
```

Comment:
brightcove, as well as all video players, needs a fixed size. also i think some of the attributes are malformed. Itworks when I set the attributes to this: (copy/pasted from the amp storybook example)

```html
    <bento-brightcove
      data-video-id="ref:amp-docs-sample"
      data-player-id="SyIOV8yWM"
      data-account="1290862519001"
      data-referrer="EXTERNAL_REFERRER"
      style="height: 100px;"
    >
    </bento-brightcove>
```